.circleci/config.yml

```diff
@@ -60,9 +48,33 @@ jobs:
           name: Run tests with Python 3.10
           command: |
             tox -e py310
+  test_feature_engine_py311:
+    docker:
+      - image: cimg/python:3.11.7
```

Comment:
@ChristopherGS any obvious idea why this is failing?

Reply:
@solegalli you need to add the test environment to the tox.ini: https://github.com/feature-engine/feature_engine/blob/main/tox.ini#L25

Tutorial: https://christophergs.com/python/2020/04/12/python-tox-why-use-it-and-tutorial/

Reply:
Life saver, thank you so much @ChristopherGS !